prometheus-ingestion-retrieval.md

```diff
@@ -0,0 +1,67 @@
+# Prometheus Ingestion
+
+IRONdb has native endpoints for accepting remote write data from a Prometheus
+installation.  Once the Prometheus module is enabled, data can be send to IRONdb 
+by setting the Prometheus remote_write endpoint to:
+
+`http://irondbnode:8112/module/prometheus/write/<accountid>/<uuid>`
+
+## Enabling Prometheus Module
+
+IRONdb must be [configured](configuration.html) such that the Prometheus module is
+enabled for reading or writing Prometheus data natively.  In the <modules> section of 
+the IRONdb config, Prometheus support is activated by including the following stanza:
+```
+<generic image="prometheus" name="prometheus"/>
+```
+Depending on installation this may be shipped enabled by default.
+
+## Namespacing
+
+Prometheus data is not namespaced by nature.  This can create confusion if different
+copies of Prometheus have identically named metrics.  Inside of IRONdb, we require 
+that all data be namespaced under a UUID.  This UUID can be arbitrarily chosen or 
+created using `uuidgen` on a typical unix system.  Each distinct set of 
+Prometheus data should have it's own UUID.   For high-availability in Prometheus it
+is the recommended pratice to have two copies collecting the same data.  While these
+two instances do not contain the same data, they do represent the same metrics, and so
+should share a common UUID for their namespace.  You may wish to send both of these
+nodes into IRONdb, where they will get merged into simply being more samples for the
+particular metrics time-streams.
+
+All metrics live under a numeric identifier (one can think of this like an
+account_id). Metric names can only be associated with an "account_id". This
+allows separate client instances that segregate queries for metric
+names, or combine them all together under a single "account_id", or even
+separate internal groups but recombining them under the client for
+visualization purposes.
```

Comment:
It looks like this was lifted from the Graphite namespacing page, but the way it's now worded doesn't make sense. I don't know enough about a typical Prometheus workflow to suggest changes, but from the outside I don't know what we're trying to say here.